DigitalLearningSolutions.Data/Services/CourseContentService.cs

```diff
@@ -8,6 +8,8 @@
     public interface ICourseContentService
     {
         CourseContent? GetCourseContent(int customisationId);
+        int GetProgressId(int candidateId, int customisationId);
+        void UpdateLoginCountAndDuration(int progressId);
     }
 
     public class CourseContentService : ICourseContentService

```

Comment:
You need to update the CourseContentServiceTests to add tests for your new methods 🙂